src/renderer/services/litecoin/common.ts

```diff
@@ -26,7 +28,12 @@ const litecoinNetwork$: Observable<ClientNetwork> = network$.pipe(
   })
 )
 
-const SOCHAIN_URL = envOrDefault(process.env.REACT_APP_SOCHAIN_URL, 'https://sochain.com/api/v2')
+const LTC_NODE_TESTNET_URL = envOrDefault(process.env.REACT_APP_LTC_NODE_TESTNET_URL, '')
+const LTC_NODE_MAINNET_URL = envOrDefault(process.env.REACT_APP_LTC_NODE_MAINNET_URL, '')
+const oLTCNodePassword = optionFromNullableString(envOrDefault(process.env.REACT_APP_LTC_NODE_PASSWORD, ''))
+const oLTCNodeUsername = optionFromNullableString(envOrDefault(process.env.REACT_APP_LTC_NODE_USERNAME, ''))
```

Comment:
Please add default values using our proxy node - with it we don't need to add ENV to GH secrets and using ENV for development only.

Reply:
just to clarify why i've set this things exactly this way for some reasons:
for URLs:
- leave a possibility for user to define his own node url via env vars (in future. this is not implemented yet)
- empty string will be processed by [xchain-litecoin client by double negation of nodeUrl parameter](https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-litecoin/src/client.ts#L68) so `xchain-litecoin` will still use its default value for appropriate network

for auth info:
- the same possibility for future for user (node operator) to define his own node's auth info

if both username and password were provided by env this info will be passed to the `xchain-litecoin` client in other case there will be passed `undefined` value and [default auth info for litecoin client will be used](https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-litecoin/src/client.ts#L74)

so with this implementation NO NEED to add anything to ENV to GH secrets

what do you think about it ?

Reply:
> empty string will be processed by xchain-litecoin client by double negation of nodeUrl parameter so xchain-litecoin will still use its default value for appropriate network

What happened if xchain-ltc will change it internal implementation?

Let's keep it simple and provide default values for now.

Reply:
👌